frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx

```diff
@@ -241,7 +248,27 @@ function CellInfoSideBar({
           <tbody>
             {Object.entries(data.marker_genes).map((gene) => (
```

Comment:
For readability, let's change `gene` to `[symbol, metadata]` or something better!